admin/load.php

```diff
@@ -531,6 +534,29 @@ function perflab_enqueue_modules_page_scripts() {
 	wp_enqueue_style( 'thickbox' );
 
 	wp_enqueue_script( 'plugin-install' );
+
+	wp_enqueue_script(
```

Comment:
We should only enqueue these scripts if needed, to avoid unnecessary JavaScript. It is only needed if there are any active modules with standalone plugins. For example call the new `perflab_get_active_modules_with_standalone_plugins()` function here and bail early if there aren't any.